assets/css/v2/busway/departures/header.scss

```diff
@@ -0,0 +1,19 @@
+.departures-header {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  font-family: Inter, sans-serif;
+  font-size: 56px;
+  font-weight: 700;
+  line-height: 67.77px;
+  text-transform: uppercase;
+  letter-spacing: 3px;
+  background-color: #cccbc8;
+  border-top: 4px solid #b2b1af;
```

Comment:
**question:** Do we have Sass/SCSS variables for colors that I should reference?

Reply:
We already have `colors.scss` which would be a logical place to put them. Currently we do a lot of duplication of values even when they're explicitly a common design token (in Figma), but I for one would be interested in moving away from that tendency and I wouldn't expect many objections.

This particular color doesn't seem to be associated with a design token. It might be worth checking Screenkit to see if a corresponding value is actually in there and this instance just isn't linked to it.

Reply:
Also handled here: a9c7110212892826b5bfea10290cc28ebd3788f9